packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -295,7 +255,7 @@ export function SelectPanel({
       return
     }
 
-    if (isLoading) {
+    if (isLoading || items.length > 0) {
```

Comment:
noticed a bug in prod where it was setting the loading screen on empty when items had already been loaded but there were no results after filtering